src/Kusto/docs/Remove-AzKustoScript.md

```diff
@@ -8,7 +8,7 @@ schema: 2.0.0
 # Remove-AzKustoScript
 
 ## SYNOPSIS
-Deletes a Kusto principalAssignment.
+Deletes a Kusto script.
```

Comment:
The synopsis is generated from swagger, manual change will be overridden in next release. Let's revert it.